tpm2_pytss/types.py

```diff
@@ -1516,7 +1517,7 @@ class TPM2B_NAME(TPM2B_OBJECT):
     pass
 
 
-class TPM2B_NV_PUBLIC(TPM2B_OBJECT):
+class TPM2B_NV_PUBLIC(TPM_OBJECT):
```

Comment:
I guess it's simpler to avoid the complex TPM2B's then handling it in the TPM2B_OBJECT class itself. Can we rename TPM2B_OBJECT to TPM2B_SIMPLE_OBJECT to better clarify that only simple, non compound TPM2B's should inherit from it.

Reply:
Sounds good, will add it to the next pusj